interfaces/extended_sqs_client_configuration.go

```diff
@@ -0,0 +1,16 @@
+package interfaces
```

Comment:
let rename it to aws_extended_s3iface
it would be friendlier for developer to import it in IDE (e.g. vscode)

Reply:
renamed to aws_extended_sqsiface